src/main/java/org/apache/commons/lang3/time/FastTimeZone.java

```diff
@@ -29,14 +29,14 @@
 
     private static final Pattern GMT_PATTERN = Pattern.compile("^(?:(?i)GMT)?([+-])?(\\d\\d?)?(:?(\\d\\d?))?$");
 
-    private static final TimeZone GREENWICH = new GmtTimeZone(false, 0, 0);
+    //private static TimeZone GREENWICH =
```

Comment:
We tend to only leave comments that are intended to help other developers, or things that are almost-ready to be finished, for instance. This one could probably be removed.